### ISSUE TITLE
Seeding a torrent that uses both ws and udp on the same tracker crashes the tracker

### ISSUE THREAD

**Issue by gmbuell** [feross/bittorrent-tracker#78]
I can't reproduce this using tracker.webtorrent.io since I can't seem to connect using udp. However, when running a local tracker with this client code:
```javascript
var torrent = client.seed(filePath, {
    announceList: [['udp://mytracker:8000',
                    'ws://mytracker:8000']]
});
```
Crashes the tracker with
```
/usr/local/lib/node_modules/bittorrent-tracker/server.js:314
        peer.socket.send(JSON.stringify({
                   ^
TypeError: Cannot read property 'send' of undefined
    at /usr/local/lib/node_modules/bittorrent-tracker/server.js:314:20
    at Array.forEach (native)
    at /usr/local/lib/node_modules/bittorrent-tracker/server.js:313:13
    at /usr/local/lib/node_modules/bittorrent-tracker/server.js:398:5
    at Swarm.announce (/usr/local/lib/node_modules/bittorrent-tracker/lib/swarm.js:23:5)
    at Server._onAnnounce (/usr/local/lib/node_modules/bittorrent-tracker/server.js:366:9)
    at Server._onRequest (/usr/local/lib/node_modules/bittorrent-tracker/server.js:353:10)
    at Server._onWebSocketRequest (/usr/local/lib/node_modules/bittorrent-tracker/server.js:289:8)
    at WebSocket.emit (events.js:110:17)
    at Receiver.ontext (/usr/local/lib/node_modules/bittorrent-tracker/node_modules/ws/lib/WebSocket.js:797:10)
```

**Comment by feross** [comment]
Thanks for the report. I think I know what's causing this.

**Comment by feross** [comment]
Okay, this is fixed in the latest bittorrent-tracker, v4.4.4.

And the UDP server is running on tracker.webtorrent.io again :)

**Comment by feross** [comment]
https://github.com/feross/bittorrent-tracker/commit/96f6fce1ab6b68fef68d035700cfd3bb3294960c
